tests/etl/test_perf_data_load.py

```diff
@@ -0,0 +1,221 @@
+import copy
+import datetime
+import json
+import time
+
+import pytest
+
+from tests.etl.test_perf_data_adapters import _verify_signature
+from tests.test_utils import create_generic_job
+from treeherder.etl.perf import store_performance_artifact
+from treeherder.model.models import Push
+from treeherder.perf.models import (PerformanceDatum,
+                                    PerformanceFramework,
+                                    PerformanceSignature)
+
+FRAMEWORK_NAME = 'cheezburger'
+MEASUREMENT_UNIT = 'ms'
+UPDATED_MEASUREMENT_UNIT = 'seconds'
+
+
+@pytest.fixture
+def datum():
+    return {
+        'job_guid': 'fake_job_guid',
+        'name': 'test',
+        'type': 'test',
+        'blob': {
+            'framework': {'name': FRAMEWORK_NAME},
+            'suites': [
+                {
+                    'name': 'cheezburger metrics',
+                    'extraOptions': ['shell', 'e10s'],
+                    'lowerIsBetter': True,
+                    'value': 10.0,
+                    'unit': MEASUREMENT_UNIT,
+                    'subtests': [
+                        {
+                            'name': 'test1',
+                            'value': 20.0,
+                            'unit': MEASUREMENT_UNIT,
+                            'lowerIsBetter': True
+                        },
+                        {
+                            'name': 'test2',
+                            'value': 30.0,
+                            'unit': MEASUREMENT_UNIT,
+                            'lowerIsBetter': False
+                        },
+                        {
+                            'name': 'test3',
+                            'value': 40.0,
+                            'unit': MEASUREMENT_UNIT,
+                        }
+                    ]
+                },
+                {
+                    'name': 'cheezburger metrics 2',
+                    'lowerIsBetter': False,
+                    'value': 10.0,
+                    'unit': MEASUREMENT_UNIT,
+                    'subtests': [
+                        {
+                            'name': 'test1',
+                            'value': 20.0,
+                            'unit': MEASUREMENT_UNIT,
+                        }
+                    ]
+                },
+                {
+                    'name': 'cheezburger metrics 3',
+                    'value': 10.0,
+                    'unit': MEASUREMENT_UNIT,
+                    'subtests': [
+                        {
+                            'name': 'test1',
+                            'value': 20.0,
+                            'unit': MEASUREMENT_UNIT
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+
+
+@pytest.fixture
+def datum_with_new_unit():
+    return {
+        'job_guid': 'fake_job_guid',
```

Comment:
Please use here a `job_guid` different from that from `datum` fixture.

Reply:
Yep.